extension-manifest-v2/package.json

```diff
@@ -49,11 +49,11 @@
   "homepage": "https://github.com/ghostery/ghostery-extension#readme",
   "dependencies": {
     "@cliqz/url-parser": "^1.1.4",
-    "@duckduckgo/autoconsent": "^3.0.4",
+    "@duckduckgo/autoconsent": "^4.0.0",
```

Comment:
I remember @philipp-classen was testing this version and found some breakage. Can you please check with him?

Reply:
Now I also cannot reproduce anymore. Looks safe to upgrade to 4.0.0.